src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/AdmissionState/AdmissionStateCellModel.swift

```diff
@@ -0,0 +1,62 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import OpenCombine
+import UIKit.UIFont
+
+final class AdmissionStateCellModel {
+
+	// MARK: - Init
+
+	init(
+		healthCertifiedPerson: HealthCertifiedPerson
```

Comment:
Should be enough to inject the the admission state, no need to inject the whole person